src/Components/SectionSearchbar/SectionSearch.css

```diff
@@ -1,8 +1,7 @@
 .Section_searchbar {
```

Comment:
Use camelCase for your classNames everywhere or snack_case everywhere to be coherent